docs/index.rst

```diff
@@ -137,6 +137,6 @@ Thanks
 Many thanks to the folks at playfire_ for releasing their
 django-dynamic-subdomains_ app, which was the inspiration for this app.
 
-.. _playfire: http://code.playfire.com/
+.. _playfire: https://playfire.com/
```

Comment:
Re:
> I wasn't sure what to do with the playfire link as they were bought by Green Man Gaming and since shut down (looking at the timeline this happened after this fork).

How about adding a link to the Internet Archive? 🙂 (I chose the first available capture before the date of the commit that originally added the link - 98b94cc81a406a62508239d272677155a2d0443f)
```suggestion
.. _playfire: https://web.archive.org/web/20110514020134/http://code.playfire.com/
```